packages/ndla-ui/src/locale/messages-nb.ts

```diff
@@ -1226,6 +1229,13 @@ const messages = {
       recentFavourites: {
         title: "Nylig lagt til i mine mapper",
         link: "Se alle mappene dine",
+        search: "Søk etter ressurser",
+        unauthorized: "Oops. Her var det tomt! Favoriser noen ressurser for å vise dem her.",
```

Comment:
Er dette henta fra designet? Favoriser blir vel ikkje heilt korrekt her.
```suggestion
        unauthorized: "Oops. Her var det tomt! Favorittmarker noen ressurser for å vise dem her.",
```